A3A/addons/core/functions/FactoryOverhaul/fn_productionGUIControls.sqf

```diff
@@ -67,20 +128,29 @@ FO_fn_GUI_lookupFactoryIDbyName = {
 };
 
 
-/* TODO
-FO_fn_GUI_lookupItemClassbyName = {
-    params ["_unlockedItemsArray","_displayName"];
+FO_fn_GUI_displayNameToCategory = {
+    params ["_selectedItem"];
+    private _className = _selectedItem call FO_fn_GUI_findWeaponClassName;
+    private _itemType = _className call BIS_fnc_itemType;
```

Comment:
A comment giving an example return value of BIS_fnc_itemType would be nice here, so it's not required to go look it up on the wiki.

Reply:
Added a comment straight from the wiki